x-pack/metricbeat/module/googlecloud/constants.go

```diff
@@ -73,3 +71,53 @@ const (
 	LabelUser     = "user"
 	LabelMetadata = "metadata"
 )
+
+// Available perSeriesAligner map
+// https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.alertPolicies#Aligner
+var AlignersMapToGCP = map[string]monitoringpb.Aggregation_Aligner{
+	"ALIGN_NONE":           monitoringpb.Aggregation_ALIGN_NONE,
+	"ALIGN_DELTA":          monitoringpb.Aggregation_ALIGN_DELTA,
+	"ALIGN_RATE":           monitoringpb.Aggregation_ALIGN_RATE,
+	"ALIGN_INTERPOLATE":    monitoringpb.Aggregation_ALIGN_INTERPOLATE,
+	"ALIGN_NEXT_OLDER":     monitoringpb.Aggregation_ALIGN_NEXT_OLDER,
+	"ALIGN_MIN":            monitoringpb.Aggregation_ALIGN_MIN,
+	"ALIGN_MAX":            monitoringpb.Aggregation_ALIGN_MAX,
+	"ALIGN_MEAN":           monitoringpb.Aggregation_ALIGN_MEAN,
+	"ALIGN_COUNT":          monitoringpb.Aggregation_ALIGN_COUNT,
+	"ALIGN_SUM":            monitoringpb.Aggregation_ALIGN_SUM,
+	"ALIGN_STDDEV":         monitoringpb.Aggregation_ALIGN_STDDEV,
+	"ALIGN_COUNT_TRUE":     monitoringpb.Aggregation_ALIGN_COUNT_TRUE,
+	"ALIGN_COUNT_FALSE":    monitoringpb.Aggregation_ALIGN_COUNT_FALSE,
+	"ALIGN_FRACTION_TRUE":  monitoringpb.Aggregation_ALIGN_FRACTION_TRUE,
+	"ALIGN_PERCENTILE_99":  monitoringpb.Aggregation_ALIGN_PERCENTILE_99,
+	"ALIGN_PERCENTILE_95":  monitoringpb.Aggregation_ALIGN_PERCENTILE_95,
+	"ALIGN_PERCENTILE_50":  monitoringpb.Aggregation_ALIGN_PERCENTILE_50,
+	"ALIGN_PERCENTILE_05":  monitoringpb.Aggregation_ALIGN_PERCENTILE_05,
+	"ALIGN_PERCENT_CHANGE": monitoringpb.Aggregation_ALIGN_PERCENT_CHANGE,
+}
+
+const (
+	DefaultAligner = "ALIGN_NONE"
+)
+
+var AlignersMapToSuffix = map[string]string{
+	"ALIGN_NONE":           ".raw",
```

Comment:
perhaps `.value` is less intimidating? I know this are raw values but users may get confused by this suffix

Reply:
Will do!

Reply:
Done.